design/batch_retrieval_design.md

```diff
@@ -288,7 +288,31 @@ The following integration tests will be implemented in Secrets Provider for K8s
 
 ## Performance tests
 
---
+The following performance tests will be run using *Jmeter*.
+These tests will be run for both the new API and the existing one, and compare the results between them.
+
+#### Benchmark
+
+| Section     | Parameter                      | Value          |
+| ----------- | ------------------------------ | -------------- |
+| Conjur      | \# of standbys                 | 2              |
+| Conjur      | \# of followers                | 1              |
+| Environment | Kubernetes                     | OpenShift 3.11 |
```

Comment:
Why is Parameter Kubernetes and the Value Openshift here? Should it be Environment with both Openshift and Kubernetes values?

Reply:
OpenShift is a type of Kubernetes deployment.

Reply:
I understand that.. but that is not what I am saying. I am asking if we are testing on both k8s and openshift. If so, that is not clear and I ask you list them both